google/cloud/aiplatform/explain/metadata/tf/v1/saved_model_metadata_builder.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.protobuf import json_format
+from typing import Any, Dict, List, Optional
+
+from google.cloud.aiplatform.compat.types import (
+    explanation_metadata_v1beta1 as explanation_metadata,
+)
+from google.cloud.aiplatform.explain.metadata import metadata_builder
+
+try:
+    import tensorflow.compat.v1 as tf
+except ImportError:
+    raise ImportError(
+        "Tensorflow is not installed and is required to load saved model. "
+        'Please install the SDK using "pip install google-cloud-aiplatform[full]"'
+    )
+
+
+class SavedModelMetadataBuilder(metadata_builder.MetadataBuilder):
+    """Metadata builder class that accepts a TF1 saved model."""
+
+    def __init__(
+        self,
+        model_path: str,
+        tags: Optional[List[str]] = None,
+        signature_name: Optional[str] = None,
+        outputs_to_explain: Optional[List[str]] = None,
+    ) -> None:
+        """Initializes a SavedModelMetadataBuilder object.
+
+        Args:
+          model_path:
+              Required. Path to load the saved model from.
```

Comment:
Is this local, GCS, or both?

Reply:
Both, updated comments for both Tf1 and 2